src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.ts

```diff
@@ -54,6 +56,8 @@ export class OsdListComponent extends ListWithDetails implements OnInit {
   reweightBodyTpl: TemplateRef<any>;
   @ViewChild('safeToDestroyBodyTpl', { static: false })
   safeToDestroyBodyTpl: TemplateRef<any>;
+  @ViewChild('deleteOsdExtraTpl')
```

Comment:
In Angular 8 we need to specify `static` on `@ViewChild`:
```suggestion
  @ViewChild('deleteOsdExtraTpl', { static: false })
```

Reply:
@tspmelo: Yes, that's why I was wondering if there was a plan to update octopus to angular 9 as well or if I should rather manually fix this line in order to stick with angular 8 (will fix this line then as suggested, thanks for taking a look :) ).